src/children.js

```diff
@@ -1,28 +1,10 @@
-import { getContext } from "./context.js";
+import childPaths from "./childPaths.js";
 
 /**
  * Get a node’s children as an array
  * @param {object | object[]} node or nodes
  * @returns {(object | string | number | boolean | null)[]}
  */
 export default function children (node) {
-	if (Array.isArray(node)) {
-		// when node is an array, flatten to avoid nested arrays of children
-		return node.flatMap(node => children(node));
-	}
-
-	let context = getContext(this);
-
-	if (!context.isNode(node)) {
-		return [];
-	}
-
-	const childProperties = context.getChildProperties(node);
-
-	if (childProperties) {
-		return childProperties.flatMap(property => node[property] ?? []);
-	}
-	else {
-		return Object.values(node).flat();
-	}
+	return childPaths.call(this, node).map(({node}) => node);
```

Comment:
I know this is technically less efficient by a factor of 2, however since the amount of child is roughly constant I think it's well worth the tradeoff since we don't want this getting out of sync with `childPaths`